dracut/30ignition/ignition-mount.service

```diff
@@ -13,7 +13,10 @@ After=ignition-disks.service
 Before=ignition-files.service
 
 # Make sure ExecStop= runs before we switch root
+# and that network is enabled for provider with a network dependency
+# like Equinix Metal (Packet)
```

Comment:
I'm a fan of more verbosity to try to explain the nuance if you don't mind. Something like:

```suggestion
# and that we order ourselves after network such that
# if networking is brought up it will still be available
# for our ExecStop= command. On some providers like Equinix
# Metal (Packet) there is a network callback sent out
# for each Ignition stage (including umount).
```